src/uucore/src/lib/mods/io.rs

```diff
@@ -0,0 +1,83 @@
+// This file is part of the uutils coreutils package.
```

Comment:
could you please document it?
As it will be part of the API like:
https://docs.rs/uucore/0.0.24/uucore/error/index.html

Reply:
thats cool. 
I added some documentation.
Thanks.